.pre-commit-config.yaml

```diff
@@ -39,15 +39,10 @@ repos:
       - id: ansible-lint
 
   - repo: https://github.com/Yelp/detect-secrets
-    rev: v1.1.0
+    rev: v1.4.0
     hooks:
       - id: detect-secrets
-        name: detect generic secrets
-        args:
-          - --baseline
-          - .secrets.baseline
-          - --exclude-secrets
-          - (dummy|admin|mp6yc0UD9e|alfresco)
```

Comment:
I would keep these in order to not be bothered IF some secret really needs to be added. That gives people an option to add secret without having to deal with secrets baseline (something to document)

Reply:
I've discovered that if you update the baseline providing exclusions they become persistent into the baseline itself